semanticdb/metap/src/main/scala/scala/meta/internal/metap/Main.scala

```diff
@@ -170,26 +170,23 @@ class Main(settings: Settings, reporter: Reporter) {
     lineIndices(line)
   }
 
+  def pprint(range: Range, doc: Option[TextDocument]): String = {
```

Comment:
Other `pprint` methods simply print to `out` instead of returning a string. How about changing this method to do the same for consistency reasons?